packages/ZbayLite/src/renderer/forms/fields/createChannelFields.ts

```diff
@@ -20,7 +20,7 @@ export const channelNameField = (name = 'channelName'): FieldData => {
         message: ChannelNameErrors.NameTooLong
       },
       pattern: {
-        value: /^[a-z0-9]+$/g,
+        value: /^[a-zA-Z0-9 ]+$/g,
```

Comment:
Maximum length is specified by `maxLength` prop of a react hook form validation rules. There's a nice utility function in the component, which translates input value to lowercase, dash-separated channel name (we display the translated channel name as a hint under the input so that the user can type anything and see the final result)